src/Bicep.Core/Configuration/ExperimentalFeaturesEnabled.cs

```diff
@@ -19,7 +19,8 @@ public record ExperimentalFeaturesEnabled(
     bool Assertions,
     bool DynamicTypeLoading,
     bool MicrosoftGraphPreview,
-    bool CompileTimeImports)
+    bool CompileTimeImports,
+    bool PublishSource)
```

Comment:
>PublishSource [](http://example.com/codeflow?start=9&length=13)

Why did we decide to make this a feature rather than a parameter on bicep publish?

Reply:
It's just temporary.  It will be a parameter eventually.

Reply:
Got it